lib/backend/etcdv3/etcdv3.go

```diff
@@ -124,6 +124,15 @@ func (c *etcdV3Client) Create(ctx context.Context, d *model.KVPair) (*model.KVPa
 		return existing, cerrors.ErrorResourceAlreadyExists{Identifier: d.Key}
 	}
 
+	v, err := model.ParseValue(d.Key, []byte(value))
+	if err != nil {
```

Comment:
this should technically never err. hence, do we panic instead?

Reply:
I think this is leftover from something else I tried and forgot to remove. Out of curiosity though, what is the benefit of throwing a panic vs returning an error here? From what I can tell, we're in trouble regardless since an error here means we successfully stored unparseable data, which would make me think the correct action to take would be to run a `Delete` and then throw the error to let the user know.

Reply:
I think you're right, a panic might be more appropriate here since I think the error message doesn't really convey "we have stored your validated info but for some reason cannot read it back".